.github/workflows/continuous-integration.yml

```
@@ -49,26 +47,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14
+      - uses: actions/setup-node@v2
       - name: Bootstrap benchmark tests
         run: |
           npm ci --ignore-scripts
           ./node_modules/.bin/lerna bootstrap --scope "@loopback/benchmark" --include-dependencies
       - name: Run benchmark tests
         run: npm test --prefix benchmark
 
-  code-lint:
+  lint:
```

Nitpick: Let's keep it as `code-lint` to differentiate it from `commit-lint`.

```suggestion
  code-lint:
```

---

Done